test/win-bats/collection.xml

```diff
@@ -1327,6 +1327,39 @@
     call :verify_line 23 x "passed: 4 / pending: 0 / failed: 0 / total: 4"
 	</case>
 
+	<!--
+		$x:xspec-uri with usage of catalog file (XML_CATALOG)
+		
+			Test value of $x:xspec-uri with file imported via XML Catalog
+	-->
+
+	<case name="$x:xspec-uri with file imported via XML Catalog (XSLT)">
+    set "SAXON_CP=%SAXON_CP%;%APACHE_XMLRESOLVER_JAR%"
+    set "XML_CATALOG=file:///%CD:\=/%/catalog/01/catalog-rewriteURI.xml"
+
+    call :run ..\bin\xspec.bat "catalog\catalog-01_xspec-uri.xspec"
+    call :verify_retval 0
+    call :verify_line 19 x "passed: 5 / pending: 0 / failed: 0 / total: 5"
+	</case>
+
+	<case name="$x:xspec-uri with file imported via XML Catalog (XQuery)">
+    set "SAXON_CP=%SAXON_CP%;%APACHE_XMLRESOLVER_JAR%"
+    set "XML_CATALOG=file:///%CD:\=/%/catalog/01/catalog-rewriteURI.xml"
+
+    call :run ..\bin\xspec.bat -q "catalog\catalog-01_xspec-uri.xspec"
+    call :verify_retval 0
+    call :verify_line 9 x "passed: 5 / pending: 0 / failed: 0 / total: 5"
+	</case>
+
+	<case name="$x:xspec-uri with file imported via XML Catalog (Schematron)">
+    set "SAXON_CP=%SAXON_CP%;%APACHE_XMLRESOLVER_JAR%"
+    set "XML_CATALOG=file:///%CD:\=/%/catalog/01/catalog-rewriteURI.xml"
```

Comment:
`XML_CATALOG` here doesn't need to be a URI, and the same applies to `test/xspec.bats`.

```suggestion
    set "XML_CATALOG=catalog\01\catalog-rewriteURI.xml"

    call :run ..\bin\xspec.bat "catalog\catalog-01_xspec-uri.xspec"
    call :verify_retval 0
    call :verify_line 19 x "passed: 5 / pending: 0 / failed: 0 / total: 5"
	</case>

	<case name="$x:xspec-uri with file imported via XML Catalog (XQuery)">
    set "SAXON_CP=%SAXON_CP%;%APACHE_XMLRESOLVER_JAR%"
    set "XML_CATALOG=catalog\01\catalog-rewriteURI.xml"

    call :run ..\bin\xspec.bat -q "catalog\catalog-01_xspec-uri.xspec"
    call :verify_retval 0
    call :verify_line 9 x "passed: 5 / pending: 0 / failed: 0 / total: 5"
	</case>

	<case name="$x:xspec-uri with file imported via XML Catalog (Schematron)">
    set "SAXON_CP=%SAXON_CP%;%APACHE_XMLRESOLVER_JAR%"
    set "XML_CATALOG=catalog\01\catalog-rewriteURI.xml"
```

Reply:
Addressed in 75a51d2facb07b24069ef197fb7cd91d7ead50ff, in both this file and `test/xspec.bats`.